### PR TITLE
Add suggest types and restrictions

### PR DESCRIPTION
**Pull request by tomsutton1984**
Added ability to restrict autocomplete results, and suggest types

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -188,13 +190,19 @@ GooglePlacesSuggest.propTypes = {
   renderSuggest: PropTypes.func,
   search: PropTypes.string,
   suggestRadius: PropTypes.number,
+  suggestTypes: _react.PropTypes.array,
```
Comment (xuopled) [inline src/index.js]: Please use `PropTypes.array`

File: src/index.js [hunk 2]
```diff
@@ -188,13 +190,19 @@ GooglePlacesSuggest.propTypes = {
...
+  suggestComponentRestrictions: _react.PropTypes.object,
```
Comment (xuopled) [inline src/index.js]: Please use `PropTypes.object`

File: src/index.js [hunk 3]
```diff
@@ -46,6 +46,8 @@ class GooglePlacesSuggest extends Component {
       input: search,
       location: new googleMaps.LatLng(0, 0),
       radius: suggestRadius,
+        types: suggestTypes,
```
Comment (xuopled) [inline src/index.js]: Please use the same indentation. Use https://github.com/sindresorhus/atom-editorconfig to match the `.editorconfig` file

**Comment by tomsutton1984** [comment]
I've updated src/index.js as per comments - and I've tried rebasing (not sure if done correctly though?).

**Comment by xuopled** [comment]
You did not rebase. I still see your four commits. Try something like https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by tomsutton1984** [comment]
I've tried that multiple times, each time I push the commits reappear.

Sorry to ask, but is it possible to rebase on merge? https://github.com/blog/2243-rebase-and-merge-pull-requests

Otherwise, please reject the PR and I'll redo on a fresh branch/single commit - my failed attempts to rebase are slowly driving me mad.

**Comment by xuopled** [comment]
Ok I did it, thank you

**Pull request merged**